pyanaconda/modules/services/installation.py

```diff
@@ -123,3 +126,77 @@ def run(self):
         for service_name in self._enabled_services:
             log.debug("Enabling service: %s.", service_name)
             util.enable_service(service_name, root=self._sysroot)
+
+
+class ConfigureSystemdDefaultTargetTask(Task):
+    """Installation task for configuring systemd default target.
+
+    Set the correct systemd default target for the target system.
+
+    We support setting either the text only "multi-user"
+    target or the graphical target called "graphical".
+    """
+
+    SUPPORTED_SYSTEMD_TARGETS = [TEXT_ONLY_TARGET, GRAPHICAL_TARGET]
+
+    def __init__(self, sysroot, default_target=None):
+        """Create a new systemd target configuration task.
+
+        :param str sysroot: a path to the root of the target system
+        :param default_target: systemd default target to be set
+        """
+        super().__init__()
+        self._sysroot = sysroot
+        # default to the text only target
+        if not default_target:
+            default_target = TEXT_ONLY_TARGET
```

Comment:
It's apparently set to "" in the module as otherwise it would always show up in the output kickstart. But I guess we could do that better - add a flag if it showed up in the input kickstart or was set via DBus, which would trigger it being written out.

Reply:
You can return the default value in the property `default_target`, if `_default_target` is not set, and generate kickstart based on the attribute `_default_target`. I think we already do something similar somewhere else.

Reply:
OK, let's do that, but that still pretty ugly. :D

Reply:
This code can be removed now.

Reply:
Removed.